Config/Schema/schema.php

```diff
@@ -27,7 +27,8 @@ public function after($event = array()) {
 		'created' => array('type' => 'datetime', 'null' => true, 'default' => null, 'key' => 'index'),
 		'url' => array('type' => 'string', 'null' => true, 'default' => null, 'collate' => 'utf8_general_ci', 'charset' => 'utf8'),
 		'ip' => array('type' => 'string', 'null' => true, 'default' => null, 'length' => 50, 'collate' => 'utf8_general_ci', 'charset' => 'utf8'),
-		'is_delete' => array('type' => 'boolean', 'null' => false, 'default' => '0'),
+		'is_delete' => array('type' => 'boolean', 'null' => false, 'default' => '0', 'key' => 'index', 'comment' => 'true if the model was deleted.'),
+		'is_soft_delete' => array('type' => 'boolean', 'null' => false, 'default' => '0', 'key' => 'index', 'comment' => 'true if version is soft deleted.'),
```

Comment:
What is the purpose of the is_delete column vs the is_soft_delete column?

Reply:
Naming is confusing. is_delete already existed, and it’s the flag that gets set to true when the model being versioned is being deleted. The versions table acts as a restorable place of makeshift soft deletes on models this behavior is attached to. 

The new soft delete feature is for the versions table itself.

Reply:
Ah ok, I figured there was something behind this one but wanted to make sure I understood. Thank you!